tests/remote-secret/image-repository-cr-image-pull-remote-secret.go

```diff
@@ -0,0 +1,230 @@
+package remotesecret
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appservice "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	image "github.com/redhat-appstudio/image-controller/api/v1alpha1"
+	rs "github.com/redhat-appstudio/remote-secret/api/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+/*
+ * Component: remote secret
+ * Description: SVPI-574 - Ensure existence of image pull remote secret and image pull secret when ImageRepository is created
+ *              SVPI-652 - Ensure existence of image push remote secret and image push secret when ImageRepository is created
+ * Note: This test covers the newer approach (ImageRepository CR) that it is not in prod yet
```

Comment:
I would say it's preferred approach, and it's already in prod.

Reply:
Thanks! I will update